packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -342,7 +364,7 @@ const CIDocument: FC<CIDocumentProps> = ({
             <Tab tabIndex={0} label={messages.metadataTabLabel}>
               {!hasError && (
                 <MetadataPane
-                  metadata={metadata}
+                  metadata={state.metadata}
```

Comment:
Issue mentions that we need to fix this for `metadata` (doing that here) and `attributes` and `relations` for invoices/POs -- where are we doing the latter?

Reply:
https://github.com/watson-developer-cloud/discovery-components/pull/13/files#diff-158d0c756b5100722b593c523edcbd52R182 This change takes care or getting attribute / relation from the `state`